lerobot/scripts/push_dataset_to_hub.py

```diff
@@ -189,9 +191,27 @@ def push_dataset_to_hub(
 
     # convert dataset from original raw format to LeRobot format
     from_raw_to_lerobot_format = get_from_raw_to_lerobot_format_fn(raw_format)
-    hf_dataset, episode_data_index, info = from_raw_to_lerobot_format(
-        raw_dir, videos_dir, fps, video, episodes, encoding
-    )
+
+    if "oxe_rlds" in raw_format:
+        # User could provide official OXE dataset name to convert it to LeRobot format
+        # the raw_format str is as such:
+        # oxe_rlds (default)
+        # oxe_rlds.bridge_orig: (with bridge_orig as oxe_dataset_name)
+        splited_raw_format = raw_format.split(".")
+        assert len(splited_raw_format) <= 2, f"Invalid raw_format: {raw_format}"
+        if len(splited_raw_format) == 2:
+            oxe_dataset_name = splited_raw_format[1]
+            print(f"Converting dataset [{oxe_dataset_name}] from 'oxe_rlds' to LeRobot format.")
+        else:
+            oxe_dataset_name = None
+
+        hf_dataset, episode_data_index, info = from_raw_to_lerobot_format(
+            raw_dir, videos_dir, fps, video, episodes, encoding, oxe_dataset_name=oxe_dataset_name
+        )
+    else:
+        hf_dataset, episode_data_index, info = from_raw_to_lerobot_format(
+            raw_dir, videos_dir, fps, video, episodes, encoding
+        )
```

Comment:
What do you think of this simplifcation?
Could you update the comments? It's not clear to me what's the use case for `oxe_rlds` versus `oxe_rlds.bridge_orig`

```suggestion
    fmt_kwgs = {
        "raw_dir": raw_dir,
        "videos_dir": videos_dir,
        "fps": fps,
        "video": video,
        "episodes": episodes,
        "encoding": encoding,
    }

    if "oxe_rlds." in raw_format:
        # Support for official OXE dataset name inside `raw_format`.
        # For instance, `raw_format="oxe_rlds"` uses the default formating (TODO what does that mean?),
        # and `raw_format="oxe_rlds.bridge_orig"` uses the brdige_orig formating
        _, oxe_dataset_name = raw_format.split(".")
        print(f"Converting dataset [{oxe_dataset_name}] from 'oxe_rlds' to LeRobot format.")
        fmt_kwgs["oxe_dataset_name"] = oxe_dataset_name
    
    hf_dataset, episode_data_index, info = from_raw_to_lerobot_format(**fmt_kwgs)
```

Reply:
Very neat! Much better to have only one call for from_raw_to_lerobot_format.

The idea here is to allow other datasets with rlds format that are not apart of OpenX to be loaded to LeRobotDatasets.

Do you think this is an important feature to have?